index.html

```diff
@@ -0,0 +1,230 @@
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
+                </div>
+                <div class="grid-items">
+                    <div class="proj-image">
+                        <h3 class="proj-text">
+                            Proffesional Art
+                            Printing Data
+                        </h3>
+                        <p class="preview-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <div class="preview-buttons">
+                            <button class="html-btn">HTML</button>
+                            <button class="css-btn">Bootstrap</button>
+                            <button class="ruby-btn">Ruby</button>
+                        </div>
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
+                </div>
+                <div class="grid-items">
+                    <div class="proj-image">
+                        <h3 class="proj-text">
+                            Proffesional Art
+                            Printing Data
+                        </h3>
+                        <p class="preview-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <div class="preview-buttons">
+                            <button class="html-btn">HTML</button>
+                            <button class="css-btn">Bootstrap</button>
+                            <button class="ruby-btn">Ruby</button>
+                        </div>
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
+                </div>
+                <div class="grid-items">
+                    <div class="proj-image">
+                        <h3 class="proj-text">
+                            Proffesional Art
+                            Printing Data
+                        </h3>
+                        <p class="preview-text">
+                            A daily selection of privately personalized reads; no accounts or sign-ups required. Has been the industry's standard.
+                        </p>
+                        <div class="preview-buttons">
+                            <button class="html-btn">HTML</button>
+                            <button class="css-btn">Bootstrap</button>
+                            <button class="ruby-btn">Ruby</button>
+                        </div>
+                    </div>
+                    <button class="see-button">
+                        See project
+                    </button>
```

Comment:
- [ ] Good Job in implementing this task, however, As part of the project requirement, Please add the Button interactions in your styles like `enable, hover, etc`, Below is the Figma design:

![Buttons Hover](https://github.com/2004-okari/MY-PORTFOLIO/assets/43528594/32f2fb0d-6ed6-4c13-be72-ce32fc5864d5)